torch/csrc/jit/passes/quantization.cpp

```diff
@@ -64,6 +64,68 @@ graph(%self, %input):
   return values;
 }
 
+c10::optional<script::Module> findChildModule(
+    Value* module_instance,
+    const script::Module& module) {
+  if (module_instance->node()->kind() == prim::GetAttr) {
+    auto child_module_name = module_instance->node()->s(attr::name);
+    auto child_module = module.find_module(child_module_name);
+    return child_module;
+  }
+  return c10::nullopt;
+}
+
+bool valueObserved(
+    Node* n,
```

Comment:
Nit: the function is called "valueObserved" but takes `Node` instead of a `Value`. It would probably be better to take value as an argument and get the corresponding node inside.

Also, the meaning of the other parameters is unclear. Even though it's not a public API, it's worth to add a comment about it.

Reply:
oh, it's node because we only have access to node here:     
`if (valueObserved(u.user, self, module, child_module_set)) {`, but I can change the function name.

Reply:
What are we observing, a node or a value? I think observing values is the correct abstraction here, and in this case I would rather suggest extracting the value from `Use` (`u.offset` will give you an the index of the value in `u.user.inputs()`), rather than rename the function to `nodeObserved` - what would it even mean if the node had several arguments and one of them was observed and the other wasn't?

Reply:
oh, I'm not changing the name to `nodeObserved`, I changed it to `callMethodOfModuleWithSameQConfig`